src/mapbox/mapbox.ts

```diff
@@ -806,14 +815,14 @@ export default class Mapbox {
       } else {
         features = [];
       }
-      const isHovering = features.length > 0;
+      const hoveringLayersCount = this._uniqueLayersCount(features);
 
-      if (!isHovering && wasHovering) {
+      if (hoveringLayersCount < hoveredLayersCount) {
```

Comment:
Is this correct, though? What if the layer count is the same but one layer is replaced by another?

Reply:
don't know if it's ok to call pointEvents twice in one callback